src/partners/dummy.ts

```diff
@@ -0,0 +1,10 @@
+import { PluginParams, PluginResult } from '../types'
+
+export async function queryDummy(
```

Comment:
Instead of a `queryDummy` partner plugin, why not just comment out the plugin from `queryEngine.ts`?

Reply:
@samholmes knows after quick chat. removing the provider will cause a crash as the db still has a providerId to ensure we still show old reports values.